src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStatic.cs

```diff
@@ -254,6 +255,17 @@ void OnSymbolEnd(SymbolAnalysisContext symbolEndContext)
                 return false;
             }
 
+            // Auto-properties (readonly, get, set) marked with an attribute have a block context callback while they don't
+            // if there is no attribute (see https://github.com/dotnet/roslyn/issues/46132).
+            // We consider that auto-property have the intent to always be instance members so we want to workaround this issue
+            // by bailing-out if this is an accessor, there are some operations in the block but they are all "none" operations.
+            if (methodSymbol.IsAccessorMethod()
```

Comment:
Can we file a Roslyn feature request to expose `IMethodSymbol.IsAutoProperty` as a public API?

Reply:
See https://github.com/dotnet/roslyn/issues/46682